skymp5-server/cpp/server_guest_lib/WorldState.cpp

```diff
@@ -345,8 +345,8 @@ bool WorldState::AttachEspmRecord(const espm::CombineBrowser& br,
 
   if (isNpc) {
     espm::LookupResult res = br.LookupById(worldOrCell);
-    uint16_t cellFlags =
-      reinterpret_cast<espm::CELL*>(res.rec)->GetData(cache).flags;
+    auto cellRecord = reinterpret_cast<espm::CELL*>(res.rec);
```

Comment:
reinterpret_cast seems to be incorrect here

it can be WRLD here. not only CELL

Reply:
change to `espm::GetData<>` ?

Reply:
espm Convert